### PR TITLE
add param to put ssl in ELB

### PR DESCRIPTION
**Pull request by ismferd**
I want to get and put ssl cert from AWS if exist
WIP

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -94,7 +96,6 @@ wimpy_aws_elb_draining_timeout: 10 # Wait a specified timeout allowing connectio
 wimpy_cross_az_load_balancing: "no" # Distribute load across all configured Availability Zones
 wimpy_aws_elb_vpc_subnets: "{{ wimpy_vpc_subnets }}"
 wimpy_aws_elb_enable_draining: True
-wimpy_aws_elb_draining_timeout: 30
```
Comment (fiunchinho) [inline defaults/main.yml:97]: We still need this, right?

File: troposphere/asg-elb-dns.py [hunk 2]
```diff
@@ -152,6 +152,16 @@ def generate_cloudformation_template():
         Type="String",
     ))
 
+    sslcertarn = template.add_parameter(Parameter(
+        "SslCertArn",
+        Type="String",
+        Default=""
+    ))
+
+    condtions = {
```
Comment (fiunchinho) [inline troposphere/asg-elb-dns.py]: I think this is not used

Comment (ismferd) [reply troposphere/asg-elb-dns.py]: conditions or sslcertarn var?

File: defaults/main.yml (hunk 1 above)
Comment (ismferd) [reply defaults/main.yml:97]: the var is repeated with 2 values 10 and 30.
I have kept it 10

File: troposphere/asg-elb-dns.py (hunk 2 above)
Comment (fiunchinho) [reply troposphere/asg-elb-dns.py]: conditions (which is actually spelled `condtions` in the code)

File: defaults/main.yml (hunk 1 above)
Comment (fiunchinho) [reply defaults/main.yml:97]: 👍

File: troposphere/asg-elb-dns.py (hunk 2 above)
Comment (ismferd) [reply troposphere/asg-elb-dns.py]: jajajaja ok, I'm going to change it...

**Review by fiunchinho — COMMENTED** [review]
Why don't we keep Wimpy simpler by forcing users to always pass the full certificate ARN, instead of passing the name and then make Wimpy look for the ARN?

**Comment by ismaelfernandezscmspain** [comment]
The human brains are not ready for ARN syntax...
But, that would simplify some things, the best thing would be to get it from cf output.
I will do the proposal changes, I will remove the logic as you comment. Make sense

File: tasks/deploy.yml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 
+- debug: msg="RANDOM"
```
Comment (fiunchinho) [inline tasks/deploy.yml]: :P

**Comment by ismaelfernandezscmspain** [comment]
I'm going to close this PR.
that's already working today.

**Pull request closed without merging**